src/DwhProvider/LocalSnowflakeProvider.php

```diff
@@ -69,11 +68,12 @@ public function createDbtYamlFiles(array $configurationNames = []): void
             }
         }
 
+        $this->setEnvVars();
```

Comment:
Posunul jsem to výš, ať z toho můžu vycházet v `src/DwhProvider/RemoteBigQueryProvider.php:85` a nemusím tam tlačit další závislost někde v parametrech